cmd/podman/start.go

```diff
@@ -147,7 +147,7 @@ func startCmd(c *cli.Context) error {
 					logrus.Errorf("unable to detect if container %s should be deleted", ctr.ID())
 				}
 				if createArtifact.Rm {
-					if rmErr := runtime.RemoveContainer(ctx, ctr, true); rmErr != nil {
+					if rmErr := runtime.RemoveContainer(ctx, ctr, true, true); rmErr != nil {
```

Comment:
I don't think this is safe at all - what if there are actual named volumes people are using, not image volumes, attached? We're risking killing people's data here

Reply:
Yes this is questionable.  We need a mechanism to know if the volume was created just for the container or not.  But I agree we should remove the volume removal.